### ISSUE TITLE
Move json to JSON-LD format

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-quickref#219]
Seems to me that there should be JSON-LD versions of WCAG here https://github.com/w3c/wai-wcag-quickref/tree/gh-pages/_data
